app/data/returning-session-data-defaults-a11y.js

```diff
@@ -66,9 +66,8 @@ module.exports = {
     }
   ],
   status: 'Draft',
-  confirmationTitle: 'Form submitted',
-  confirmationNext:
-    "We've sent you an email to confirm we have received your form.",
+  confirmationTitle: 'Your form has been submitted',
```

Comment:
Is this `confirmationTitle` still used anywhere? If not I think you can remove it from this and the other data files.

Reply:
Good call `confirmationTitle` is now hardcoded in the template itself. Will update

Reply:
it is used in `pageTitle` for pages to set title in the `<head>`. But I've harcoded the 'Your form has been submitted' value in the actual `panel__title` in both Confirmation page previews. 

Maybe to make it more consistent I'll use `{{ data['confirmationTitle'] }}` instead of harcoding the message which will make it less places to update the code if anything changes 😃